style.css

```diff
@@ -0,0 +1,877 @@
+  line-height: 2.4rem;
+}
+
+@media screen and (min-width: 766px) {
+  .header-content {
+    height: 65vh;
+    padding-left: 0.5em;
+  }
+
+  h1 {
+    font-size: 2.5rem;
+    font-weight: 900;
+  }
+
+  .abb-info {
+    background-image: url('asset/about-dektop.png');
+    width: 26rem;
+  }
+
+  .about-info {
+    max-width: 40rem;
+    margin: 0 auto;
+  }
+
+  .date {
+    font-size: 1.8rem;
+  }
+}
+
+/* $-Program-$ */
+.program {
+  flex-direction: column;
+  padding: 1rem;
+  z-index: -100;
+  opacity: 1;
+  background: radial-gradient(circle, transparent 20%, #1a1a1e 20%, #1a1a1e 80%, transparent 80%, transparent), radial-gradient(circle, transparent 20%, #1a1a1e 20%, #1a1a1e 80%, transparent 80%, transparent) 17.5px 17.5px, linear-gradient(#000 1.4000000000000001px, transparent 1.4000000000000001px) 0 -0.7000000000000001px, linear-gradient(90deg, #000 1.4000000000000001px, #1a1a1e 1.4000000000000001px) -0.7000000000000001px 0;
+  background-size: 35px 35px, 35px 35px, 17.5px 17.5px, 17.5px 17.5px;
+  background-color: #1a1a1e;
+  color: var(--tetiary-color);
+}
+
+h2 {
+  text-align: center;
+  display: block;
+}
+
+h2::after {
+  width: 4rem;
+  height: 2px;
+  background-color: #ec5342ac;
+  content: '';
+  display: block;
+  margin: 1rem auto;
+}
+
+.program-items {
+  flex-direction: column;
+}
+
+.p-cards {
+  align-items: center;
+  background-color: #d3d3d32f;
+  padding: 2rem;
+  border-radius: 4px;
+}
+
+.p-icon {
+  width: 3rem;
+  border-radius: 5px;
+  background-color: var(--tetiary-color);
+}
+
+.dp-text {
+  width: 9rem;
+}
+
+.dp-summary {
+  width: 100%;
+}
+
+.p-summary {
+  font-size: 12px;
+  font-weight: 600;
+  text-align: left;
+}
+
+.program-btn {
+  margin: 2rem auto;
+  padding: 1.5rem;
+  background-color: var(--secondary-color);
+  color: var(--tetiary-color);
+  font-weight: 600;
+  transition: 0.7s ease-in-out;
+}
+
+.program-btn:hover {
+  color: white;
+  border-radius: 20px;
+  transform: scale(1.1);
+}
+
+.program-text {
+  text-align: center;
+  text-decoration: underline;
+  font-size: small;
+  display: none;
+  margin: 3rem;
+}
+
+@media screen and (max-width: 300px) {
+  .p-cards {
+    flex-direction: column;
+    text-align: center;
+  }
+}
+
+@media screen and (min-width: 766px) {
+  .program-items {
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    flex-wrap: wrap;
+  }
+
+  .p-cards {
+    flex-direction: column;
+    text-align: center;
+    padding: 1rem;
+    margin: 0.2rem;
+    width: 8rem;
+    height: 14rem;
+  }
+
+  .program-btn {
+    display: none;
+  }
+
+  .program-text {
+    display: block;
+  }
+}
+
+/* $-Coaches-$ */
+.coaches {
+  padding: 1rem;
+  flex-direction: column;
+}
+
+.coach-cards {
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.c-card {
+  max-width: 20rem;
+  width: 100%;
+}
+
+.c-images {
+  position: relative;
+  width: 6rem;
+  height: 6rem;
+}
+
+.c-summary {
+  font-size: 10px;
+  font-weight: 600;
+}
+
```

Comment:
- [ ] [REQUIRED] Please, your layout of `Coach cards` should like a [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)

![image](https://user-images.githubusercontent.com/63915024/190243520-3b3c10a2-659f-4dab-abaa-b9abc5ae4a6e.png)

Reply:
- [ ] [REQUIRED] You must fulfill the [requirement before](https://github.com/UsirusenIkon/Allblack-Basketball-Camp-First-Capstone-/pull/1/files#r971210020) me because the content of this module is `"HTML&CSS"` which means you must fully respect the [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) provided by `Microverse`.